src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/backpacks/SlimefunBackpack.java

```diff
@@ -65,6 +66,12 @@ public boolean isItemAllowed(@Nonnull ItemStack item, @Nullable SlimefunItem ite
         if (SlimefunTag.SHULKER_BOXES.isTagged(item.getType())) {
             return false;
         }
+        if (SlimefunItem.getByID(getId()) instanceof HazmatBackpack && !(itemAsSlimefunItem instanceof Radioactive)) {
+            return false;
+        }
+        if (itemAsSlimefunItem instanceof Radioactive && !(SlimefunItem.getByID(getId()) instanceof HazmatBackpack)) {
```

Comment:
`(SlimefunItem.getByID(getId())`

WTF...

Reply:
getId returns the id of the backpack

Reply:
you just got the id from the item then got the item from the id... loopy loopy